recipes/yaqd-fakes/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "yaqd-fakes" %}
+{% set version = "2020.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 052d13a7064c7537d0188dfa2c2d6613b4e42ae9defa4e1d170532ef9d350ddf 
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - yaqd-fake-continuous-hardware = yaqd_fakes._fake_continuous_hardware:FakeContinuousHardware.main
+    - yaqd-fake-discrete-hardware = yaqd_fakes._fake_discrete_hardware:FakeDiscreteHardware.main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - flit
+    - python >=3.7
+    - yaqd-core >=2020.05.1
```

Comment:
Is this really need to create the package?

Reply:
yes, [flit](https://flit.readthedocs.io/en/latest/index.html) needs all dependencies at build time

Reply:
That is annoying